plugin-server/src/utils/db/utils.ts

```diff
@@ -30,12 +30,13 @@ export function sanitizeEventName(eventName: any): string {
 
 export function timeoutGuard(
     message: string,
-    context?: Record<string, any>,
+    context?: Record<string, any> | (() => Record<string, any>),
```

Comment:
why this change? I didn't see it being used anywhere?

Reply:
Called by the run function. Means we can generate it _only_ when the log is actually executed. Currently we call JSON.stringify() on every event for no reason